var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -86,6 +86,8 @@ class Openblas(MakefilePackage):
           sha256='f1b066a4481a50678caeb7656bf3e6764f45619686ac465f257c8017a2dc1ff0',
           when='@0.3.0:0.3.3')
 
```

Comment:
Can you add a comment pointing to the PR that this patch originates from?

Reply:
Oh weird, I did but it somehow vanished. Adding now.